spec/event_source/subscribers/fdsh/insurance_policies/posted_subscriber_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'shared_examples/family_response3'
+
+RSpec.describe Subscribers::Fdsh::InsurancePolicies::PostedSubscriber, dbclean: :after_each do
+  include_context "family response"
+
+  let(:logger) { instance_double("::Logger", info: nil, debug: nil, warn: nil, error: nil) }
+
+  # Please refactor this test once we move to logic for multiple assistance years into an operation
+  describe 'process_insurance_policies_posted_event_for_h36' do
+    context "prospective assistance_year" do
```

Comment:
Also, can you update the tests to verify the creation of the transaction(for the given policy) for the expected transmission for the cases where the insurance policy is for the past year, current year, and prospective year?